src/components/containers/ProfileSidebar/ProfileSidebar.jsx

```diff
@@ -0,0 +1,74 @@
+/* eslint-disable no-console */
+/* eslint-disable react/prop-types */
+import React, { Component } from 'react';
+import { Icon } from 'semantic-ui-react';
+
+import Avatar from '../../presentationals/Avatar/Avatar.jsx';
+import Button from '../../presentationals/Button/Button.jsx';
+
+import './ProfileSidebar.scss';
+
+class ProfileSidebar extends Component {
+  constructor(props) {
+    super(props);
+  }
+  render() {
+    let userAvatar = {
+      url:
+        this.props.profile.image ||
+        'https://res.cloudinary.com/nedy123/image/upload/v1557149386/avatar_pupqrh.svg',
+      class: 'avatar-xl',
+    };
+
+    const CameraIcon = () => <Icon name="camera" />;
+    const OverlayImage = () => (
+      <div className="overlay-image">
+        <CameraIcon />
+      </div>
+    );
+
+    const { followers, followings, username, email } = this.props.profile;
+    return (
+      <div className="sidebar">
+        <div className="avatar-div">
+          <Avatar avatar={userAvatar} />
```

Comment:
We can rewrite the Avatar` component to use its props directly as shown below
```javascript 
<Avatar 
    url={} 
    className={} 
/>

Reply:
I have rewritten that.